core/mixins/thumbnail.js

```diff
@@ -1,15 +1,22 @@
-import { getThumbnailPath as _thumbnailHelper } from '@vue-storefront/core/helpers'
+import { getThumbnailPath } from '@vue-storefront/core/helpers'
 
 export const thumbnail = {
   methods: {
     /**
-     * Return thumbnail URL for specific base url
-     * @param {String} relativeUrl
-     * @param {Int} width
-     * @param {Int} height
+     * Return thumbnail URL for specific base url and path
+     * @param {string} relativeUrl
+     * @param {number} width
+     * @param {number} height
+     * @param {string} pathType
      */
-    getThumbnail (relativeUrl, width, height) {
-      return _thumbnailHelper(relativeUrl, width, height)
-    }
+    getThumbnail: (relativeUrl, width, height, pathType) => getThumbnailPath(relativeUrl, width, height, pathType),
+
+    /**
+     * Return thumbnail URL for specific base url using media path
+     * @param {string} relativeUrl
+     * @param {number} width
+     * @param {number} height
+     */
+    getMediaThumbnail: (relativeUrl, width, height) => getThumbnailPath(relativeUrl, width, height, 'media')
```

Comment:
Shouldn't we have some kind of default config for it other wise i don't know how this can ever function. 

So like this: `"media": "/"`?

Reply:
In the docs we tell that the you have to remove `/catalog/product` from the API's `magento1.imgUrl` or `magento2.imgUrl` config property so whats then left will be `http://magento-demo.local/media` as default value and "root-path". That's why we don't need `"media": "/"` anymore.

Reply:
Yea your are right I was probably only a bit confused that we use the media pathType which is not set in the config. :smile: